# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/e2fsprogs/hotfix-llseek.patch.musl
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- e2fsprogs-1.47.0/lib/ext2fs/llseek.c.vanilla	2023-12-20 11:11:58.021667278 +0100
+++ e2fsprogs-1.47.0/lib/ext2fs/llseek.c	2023-12-20 11:14:21.355661160 +0100
@@ -63,7 +63,7 @@
 #define __NR__llseek            140
 #endif
 
-#ifndef __i386__
+#if 0 // ndef __i386__
 static int _llseek (unsigned int, unsigned long,
 		   unsigned long, ext2_loff_t *, unsigned int);
 
@@ -77,7 +77,7 @@
 	ext2_loff_t result;
 	int retval;
 
-#ifndef __i386__
+#if 0 // ndef __i386__
 	retval = _llseek(fd, ((unsigned long long) offset) >> 32,
 #else
 	retval = syscall(__NR__llseek, fd, (unsigned long long) (offset >> 32),
